controllers/decks-controllers.js

```diff
@@ -0,0 +1,11 @@
+const { fetchDecks } = require("../models/decks-models");
+
+exports.getDecks = (req, res, next) => {
+  fetchDecks()
```

Comment:
in terms of naming conventions should we kind of define names for get/post/patch when doing operations with the models ? I think we should keep fetch whenever we do get operations to the db  models :)